t/plugin/jwt-auth.t

```diff
@@ -148,7 +148,14 @@ qr/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.\w+.\w+/
 
 
 
-=== TEST 6: verify, missing token
+=== TEST 6: test for unsupport method
```

Comment:
`unsupported` would be better

Reply:
I agree with you, and I have corrected the description